addons/portal/wizard/portal_wizard_views.xml

```diff
@@ -22,7 +22,7 @@
             <field name="name">Grant portal access</field>
             <field name="model">portal.wizard</field>
             <field name="arch" type="xml">
-                <form string="Portal Access Management">
+                <form string="Portal Access Management" js_class="portal_wizard_user_form">
```

Comment:
Yes, It is done. Good exercise to discover owl :) . However the rendering has changed quite a bit with owl, so the tooltip service does not show tooltips on disabled buttons as before. I've set an action to be used if clicked, the same as the one returned after portal actions. It will simply refresh the modal. It makes the diff small and should not be an issue, I believe? The tooltips are necessary and this method prevents - even - more js. What do you think?